docs/pipelines/ner/bmi.md

```diff
@@ -0,0 +1,48 @@
+# BMI
+The `eds.bmi_rule` pipeline extracts the Body Mass Index of a patient.
+This pipeline simply:
+1. looks for some patterns:
+      - `'bmi'`
+      - `'body mass index'`
+      - `'imc'`
+      - `'indice (de) masse corporelle'`
+2. tries to extract a number from the text after the patterns
+
+
+# Height and Weight
+The pipelines `eds.bmi_height` and `eds.bmi_weight`
+extract the height and the weight of a patient.
+They can be used to compute the BMI.
+The values are normalized automatically (e.g. `'180cm' -> 1.8`, `'6kg500' -> 6.5`).
+
+# Usage
+The name of the matcher is available with `e._.matcher_name`.
+The value of the extracted metric is available as a float in `e._.metric_value`.
+
+```python
+nlp = spacy.blank("fr")
+nlp.add_pipe(
+    "eds.normalizer",
+    config=dict(
+        lowercase=True,
+        accents=True,
+        quotes=True,
+        pollution=False,
+    ),
+)
+nlp.add_pipe("eds.sentences")
+nlp.add_pipe("eds.weight")
+nlp.add_pipe("eds.height")
+nlp.add_pipe("eds.bmi_rule")
+
+text = "le patient a un imc de 23.4 kg.m-2."
+doc = nlp(text)
+e = doc.ents[0]
+
+
+e._.matcher_name
+# Out: 'eds.bmi_rule'
+
+e._.metric_value
+# Out: 23.4
+```
```

Comment:
Use the documentation for `negation` for inspiration. For instance: add the configuration...